server/controller/FlightPhotoController.js

```diff
@@ -20,12 +20,18 @@ const {
 } = require("./Validation");
 const multer = require("multer");
 
-const { createThumbnail, deleteImages } = require("../helper/ImageUtils");
+const {
+  createThumbnail,
+  deleteImages,
+  resizeImage,
+} = require("../helper/ImageUtils");
 const logger = require("../config/logger");
 
 const IMAGE_STORE = process.env.SERVER_DATA_PATH + "/images/flights";
 const THUMBNAIL_IMAGE_HEIGHT = 310;
 const MAX_PHOTOS = 8;
+const IMAGE_BYTES_LIMIT = 1_500_000;
+const IMAGE_HEIGHT_LIMIT = 1_200;
```

Comment:
Das finde ich etwas wenig. Wie wäre mit max 3mb und 2160p (4k) Und warum nur nach image height limit? was ist ein Hochformat?

Reply:
War ein reines Bauchgefühl.
Können das gerne größer machen.

Die Breite bleibt proportional, wenn ich die Höhe anpasse.

Da die meisten Menschen ihren Monitor im Querformat haben, interessiert mich da nur die Höhe. Das Bild sollte halt einmal ganz in die Höhe des Monitors passen.

4k sehe ich da aber generell etwas kritisch. Die wenigsten haben Geräte (ja ich weiß Retina...) die das unterstützen. Wir müssen allerdings den Speicherplatz dafür bereitstellen.

Reply:
und was machen wir in fünf Jahren? das gleiche problem haben wir ja jetzt schon mit den alten xccup Bildern, oder?